tests/Unit/Service/DraftsServiceTest.php

```diff
@@ -373,54 +373,6 @@ public function testConvertToOutboxMessage(): void {
 		$this->draftsService->updateMessage($account, $message, $to, $cc, $bcc, $attachments);
 	}
 
-	public function testConvertToOutboxMessageNoRecipients(): void {
```

Comment:
right, the test makes no sense anymore because we just dropped the implicit conversion :+1: